### DISCUSSION TITLE
unknown bug

### DISCUSSION THREAD

**Discussion by muhaimincs** [planetscale/discussion#91] [Bug Reports] [answered]
what does this mean?

```js
{
  code: 'ER_UNKNOWN_ERROR',
  errno: 1105,
  sqlMessage: 'vttablet: rpc error: code = InvalidArgument desc = missing bind var TABLE_NAME (CallerID: unsecure_grpc_client)',
  sqlState: 'HY000',
  index: 0
}
```

**Comment by jgreet** [comment, marked as answer]
That's an error from [Vitess](https://vitess.io/), the clustering system we use (and maintain) to manage databases in our infrastructure. 

Could you show us the SQL that's producing the error? It would also be useful to know which database this is. Feel free to submit that information at https://planetscale.com/support if you don't want to share it here.

**Reply by muhaimincs** [reply]
ok I'll contact support

**Reply by jgreet** [reply]
Thanks. Got your email, we're looking into it.

**Reply by jgreet** [reply]
We have a fix for the problem in Vitess: https://github.com/vitessio/vitess/pull/8771

I'll update this thread when we have that fix rolled out on PlanetScale, it could take a couple of weeks.

**Reply by jgreet** [reply]
The fix from https://github.com/vitessio/vitess/pull/8771 is live on PlanetScale now.
